src/Microsoft.Data.Analysis.Interactive/Microsoft.Data.Analysis.Interactive.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
Tabs hurt my eyes.

Reply:
ugh, this setting must be mis-matched locally on my machine between VSCode and VS :\ Will fix in a bit

Reply:
Out of curiosity, how did you see tabs in this file in GH? I changed all tabs to spaces (no idea how tabs occurred in the first place) in 3 csprojs in the PR, but unfortunately there is no "Change all tabs to spaces in all files" in VS I think. For now, a solution wide regex search yields no tabs in source code

Reply:
It looked too indented then I saw it while trying to select white space.

Reply:
Has codeflow been fixed to work better with Github comment threads?  That was a deal breaker in the past.

Reply:
Yeah, it still has some issues with comment threads. Typically I only create new conversations from CodeFlow. And respond to threads in GH. I usually have them both open on 2 different windows. I just like seeing the full file, searching across the change, seeing the tree view of files, etc in CodeFlow. It's a much better experience for me.